conans/test/integration/package_info_test.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+
+from conans.paths import CONANFILE, CONANFILE_TXT
+from conans.test.utils.tools import TestClient
+
+
+class TestPackageInfo(unittest.TestCase):
+
+    def package_info_called_in_local_cache_test(self):
+        client = TestClient()
+        conanfile_tmp = '''
+from conans import ConanFile
+from conans.tools import mkdir
+import os
+
+class HelloConan(ConanFile):
+    name = "%s"
+    version = "1.0"
+    build_policy = "missing"
+    options = {"switch": ["1", "0"]}
+    default_options = "switch=0"
+    %s
+    
+    def build(self):
+        self.output.warn("Env var MYVAR=" + os.getenv("MYVAR", ""))
+
+    def package_info(self):
+        self.output.warn("Info from " + self.name)
```

Comment:
Maybe check this output? Why if not used?

Reply:
Yes, it's from a previous version of the test. Now it's useless